examples/snippets/batch/gettingStartedErrorHandling.ts

```diff
@@ -22,13 +22,13 @@ const recordHandler = async (record: SQSRecord): Promise<void> => {
     const item = JSON.parse(payload);
     logger.info('Processed item', { item });
   } else {
-    // prettier-ignore
+    // biome-ignore format:
```

Comment:
```suggestion
    // biome-ignore format: we need the comment in the next line to stay there to annotate the code snippet in the docs
```

Reply:
Could you also add the same "explanation" to the other instances where you used `biome-ignore`? Thanks!